pkg/gofr/http/request_test.go

```diff
@@ -272,3 +275,45 @@ func TestBind_FormURLEncoded(t *testing.T) {
 		t.Errorf("Bind error. Got: %v", x)
 	}
 }
+
+func TestGetAuthInfo_BasicAuth(t *testing.T) {
+	req := httptest.NewRequest(http.MethodGet, "/", http.NoBody)
+
+	ctx := context.WithValue(req.Context(), middleware.Username, "validUser")
+	*req = *req.Clone(ctx)
+	gofrRq := NewRequest(req)
+
+	u := gofrRq.GetAuthInfo()
+
+	assert.Equal(t, "validUser", u["Username"])
+}
+
+func TestGetAuthInfo_ApiKey(t *testing.T) {
+	req := httptest.NewRequest(http.MethodGet, "/", http.NoBody)
+
+	ctx := context.WithValue(req.Context(), middleware.APIKey, "9221e451-451f-4cd6-a23d-2b2d3adea9cf")
+	*req = *req.Clone(ctx)
+	gofrRq := NewRequest(req)
+
+	u := gofrRq.GetAuthInfo()
+
+	assert.Equal(t, "9221e451-451f-4cd6-a23d-2b2d3adea9cf", u["ApiKey"])
```

Comment:
```suggestion

```suggestion
	assert.Contains(t, u, "ApiKey")
	assert.Equal(t, "9221e451-451f-4cd6-a23d-2b2d3adea9cf", u["ApiKey"])
```